coderd/provisionerdserver/provisionerdserver_test.go

```diff
@@ -384,6 +454,92 @@ func TestUpdateJob(t *testing.T) {
 		require.NoError(t, err)
 		require.Equal(t, "# hello world", version.Readme)
 	})
+
+	t.Run("TemplateVariables", func(t *testing.T) {
+		t.Parallel()
+
+		t.Run("Valid", func(t *testing.T) {
+			srv := setup(t, false)
+			job := setupJob(t, srv)
+			version, err := srv.Database.InsertTemplateVersion(ctx, database.InsertTemplateVersionParams{
```

Comment:
Might be a good idea to use per-test timing out context in these?

Reply:
Fixed